test_reminder.py

```diff
@@ -0,0 +1,13 @@
+from datetime import date
+
+# import pytest
+import pytz
+
+from reminder import Reminder
+
+timezone = pytz.timezone("Europe/Zurich")
+
+
+def test_get_monday():
+    test_date = date(2023, 6, 15)
+    assert Reminder().get_monday(test_date) == date(2023, 6, 12)
```

Comment:
This is a very good start, the problem is if this runs, for example next week, it will return different result. For fixing this issue we should mock the `now` date and to do that we can use a module like [freezegun](https://github.com/spulec/freezegun) and you can decorate your test like the following:

```python
from freezegun import freeze_time

@freeze_time("2023-06-15")
def test_get_monday():
    test_date = date(2023, 6, 15)
    assert Reminder().get_monday(test_date) == date(2023, 6, 12)
```

Reply:
I pushed another commit with this change. I have to write the test for 'reminder_next' and 'notify_daily_meeting_now'